xds/internal/xdsclient/bootstrap/bootstrap.go

```diff
@@ -162,6 +162,8 @@ func (sc *ServerConfig) UnmarshalJSON(data []byte) error {
 			} else {
 				logger.Warningf("Failed to build credentials bundle for %q, error: %v", cc.Type, err)
 			}
+		} else {
+			logger.Warningf("Credential %q is not registered", cc.Type)
```

Comment:
Oh this I feel like is definitely not a warning; maybe info, but even that feels unnecessary.  If there are no registered credentials at all, though, what do we do?  We just have no credentials and the client just fails to be created as a result?  I think we could do better error checking (this is a pre-existing problem) unless I'm missing something else.  @menghanl?

Reply:
Hmm, yeah, we probably don't need to log anything for unregistered creds. The same as in balancers (we don't log if the specified balancer is not registered).
And if none of the specified creds is found, the bootstrap parsing will fail later due to no creds.

Reply:
Removed the log when the credential is not found.